app/src/main/java/com/concordium/wallet/ui/bakerdelegation/common/DelegationBakerViewModel.kt

```diff
@@ -784,4 +799,9 @@ class DelegationBakerViewModel(application: Application) : AndroidViewModel(appl
 
     fun getTempFileWithPath(): Uri =
         Uri.parse("content://" + BuildConfig.PROVIDER_AUTHORITY + File.separator.toString() + FILE_NAME_BAKER_KEYS)
+
+    fun getAvalibleBalance() = bakerDelegationData.account?.getAtDisposalWithoutStakedOrScheduled(
```

Comment:
There is a small typo here:
```suggestion
    fun getAvailableBalance() = bakerDelegationData.account?.getAtDisposalWithoutStakedOrScheduled(
```